app/jobs/import_from_vacancy_sources_job.rb

```diff
@@ -1,12 +1,12 @@
 class ImportFromVacancySourcesJob < ApplicationJob
   SOURCES = [
-    VacancySource::Source::Ark,
-    VacancySource::Source::Broadbean,
-    VacancySource::Source::Every,
-    VacancySource::Source::Fusion,
-    VacancySource::Source::MyNewTerm,
-    VacancySource::Source::UnitedLearning,
-    VacancySource::Source::Ventrus,
+    Vacancies::Import::Sources::Ark,
+    Vacancies::Import::Sources::Broadbean,
+    Vacancies::Import::Sources::Every,
+    Vacancies::Import::Sources::Fusion,
+    Vacancies::Import::Sources::MyNewTerm,
+    Vacancies::Import::Sources::UnitedLearning,
+    Vacancies::Import::Sources::Ventrus,
```

Comment:
This is the main change. 

The PR changes are to match this namespacing structure within the `app/services` folder.